pkg/action/common.go

```diff
@@ -39,3 +42,15 @@ func ChrootHook(config *config.Config, hook string, chrootDir string, bindMounts
 	}
 	return utils.ChrootedCallback(config, chrootDir, bindMounts, callback)
 }
+
+func createExtraDirsInRootfs(cfg *config.Config, extradirs []string, target string) {
+	for _, d := range extradirs {
+		if exists, _ := fsutils.Exists(cfg.Fs, filepath.Join(target, d)); !exists {
+			cfg.Logger.Debugf("Creating extra dir %s under %s", d, target)
+			err := cfg.Fs.Mkdir(filepath.Join(target, d), cnst.DirPerm)
```

Comment:
Given the Fs is passed through the config, it might be easy to mock it and create a test for this new functionality.

Sorry for the late review, feel free to skip or do it in another PR :)

Reply:
umm good point, will do!